types/openapi-factory/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": false,
```

Comment:
Why no strictFunctionTypes?

Reply:
I was pretty sure i set that back to true, must have missed it, please excuse me